addons/website_sale/static/src/js/website_sale_reorder.js

```diff
@@ -148,9 +148,11 @@ export class ReorderDialog extends Component {
         product.combinationInfo = await this.rpc("/sale/get_combination_info_website", {
             product_template_id: product.product_template_id,
             product_id: product.product_id,
-            combination: [],
+            combination: product.combination || [],
             add_qty: product.qty,
             pricelist_id: false,
+            no_variant_attribute_values: product.no_variant_attribute_values,
```

Comment:
@Feyensv you have forgot to passe this, it is necessary to update the price if you update the qty.

Reply:
Why ? the no_variant attribute values are given through `combination` (and `no_variant_attribute_values` is not an expected parameter of `get_combination_info`)
The price will be the correct one.